client/blocks/post-item/index.jsx

```diff
@@ -1,35 +1,51 @@
 /**
  * External dependencies
  */
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { isEnabled } from 'config';
 import { getEditorPath } from 'state/ui/editor/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
 import { getNormalizedPost } from 'state/posts/selectors';
+import { getSite, getSiteTitle } from 'state/sites/selectors';
+import SiteIcon from 'blocks/site-icon';
 import Card from 'components/card';
 import PostRelativeTime from 'blocks/post-relative-time';
 import PostStatus from 'blocks/post-status';
 import PostTypeListPostThumbnail from 'my-sites/post-type-list/post-thumbnail';
 import PostActionsEllipsisMenu from 'my-sites/post-type-list/post-actions-ellipsis-menu';
 import PostTypePostAuthor from 'my-sites/post-type-list/post-type-post-author';
 
-function PostItem( { translate, globalId, post, editUrl, className, compact } ) {
+function PostItem( { translate, globalId, post, site, editUrl, siteTitle, isAllSitesModeSelected, className, compact } ) {
 	const title = post ? post.title : null;
-	const classes = classnames( 'post-item', className, {
+	const postItemClasses = classnames( 'post-item', className, {
 		'is-untitled': ! title,
 		'is-mini': compact,
 		'is-placeholder': ! globalId
 	} );
 
+	const isSiteVisible = isEnabled( 'posts/post-type-list' ) && isAllSitesModeSelected;
+	const titleMetaClasses = classnames( 'post-item__title-meta', { 'site-is-visible': isSiteVisible } );
```

Comment:
Good catch! Yes, it's and `is` modifier.

I just started working on another feature which requires some refactoring of this part of the code, and this modifier will disappear, or it will be renamed at the very least so I'll take care of this issue there.